lib/membrane_dashboard/dagre.ex

```diff
@@ -61,20 +61,51 @@ defmodule Membrane.Dashboard.Dagre do
 
```

Comment:
I think it will be nice to update module docs by adding info about new/existing/dead elements and that they have different colors.

Reply:
I will add the information in the `G6Marshaller` as the `Dagre` itself should not be aware of the fact that the nodes are coloured.